src/pages/workspace/workflows/approvals/ApprovalWorkflowEditor.tsx

```diff
@@ -116,16 +152,22 @@ function ApprovalWorkflowEditor({approvalWorkflow, policyID}: ApprovalWorkflowEd
 
                 <MenuItemWithTopDescription
                     description={translate('workflowsCreateApprovalsPage.additionalApprover')}
-                    onPress={() =>
-                        Navigation.navigate(
-                            ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_APPROVER.getRoute(policyID, approvalWorkflow.approvers.length, ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_NEW.getRoute(policyID)),
-                        )
-                    }
+                    onPress={addAdditionalApprover}
                     shouldShowRightIcon
                     wrapperStyle={styles.sectionMenuItemTopDescription}
                     errorText={approvalWorkflow?.errors?.additionalApprover ? translate(approvalWorkflow.errors.additionalApprover) : undefined}
                     brickRoadIndicator={approvalWorkflow?.errors?.additionalApprover ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
                 />
+
+                {removeApprovalWorkflow && !approvalWorkflow.isDefault && (
```

Comment:
I think we should add a new props called `shouldDisplayDeleteButton` and pass this prop from parent component

```
const shouldDisplayDeleteButton = approvalWorkflow.action === EDIT && !approvalWorkflow.isDefault
```

Reply:
Either way I'll provide both `removeApprovalWorkflow` and `approvalWorkflow` props from the parent, so let's call it a NAB and leave as is